compiler/qsc_partial_eval/src/evaluation_context.rs

```diff
@@ -166,12 +166,12 @@ impl Scope {
     }
 
     /// Gets a mutable variable.
-    pub fn find_mutable_var(&self, var_id: VariableId) -> Option<&MutableVar> {
+    pub fn find_mutable_kind(&self, var_id: VariableId) -> Option<&MutableKind> {
```

Comment:
Why are all these changes part of `add prereqs check and docs for building on mac`?

Reply:
Because they get hit when updating to rust 1.79. While I was here, I fixed it up so that we can update to 1.79 at any time.